test/e2e/cypress/integration/recents-panel.spec.ts

```diff
@@ -0,0 +1,19 @@
+describe('Recents panel', () => {
+    beforeEach(() => {
+      cy.clearLocalStorage();
+      cy.login();
+      cy.visit('/app/recents');
+    });
+
+    it('Should list recents', () => {
+      cy.contains('example.txt');
+    });
+
+    it('Should get link', () => {
+      cy.get('button.file-list-item-actions-button').click();
+      cy.get('a').contains('Get link').click();
```

Comment:
Here we want to make sure the link is generated, try to find out if there is a way in cypress to check if something has been added to clipboard and also that the pop up saying "share link copied to clipboard" appears